test/y2security/autoinst_profile/security_policy_section_test.rb

```diff
@@ -41,4 +41,55 @@
       end
     end
   end
+
+  describe ".new_from_system" do
+    let(:service_enabled?) { true }
+
+    before do
+      allow(CFA::SsgApply).to receive(:load).and_return(file)
+      allow(Yast::Service).to receive(:enabled?).and_return(service_enabled?)
+    end
+
+    context "when the ssg-apply service does not exist" do
+      let(:file) { instance_double(CFA::SsgApply, empty?: true) }
+
+      it "returns an empty section" do
+        section = described_class.new_from_system
+        expect(section.to_hashes).to be_empty
+      end
+    end
+
+    context "when the ssg-apply service exists is disabled" do
```

Comment:
NP: Would suggest something like, but as this is a merge PR just merge it as it is...

```suggestion
    context "when the ssg-apply service exists but it is disabled" do
```